### PR TITLE
Bring together text and console-text transformers

### PR DESCRIPTION
**Pull request by lgeiger**

### CODE REVIEW

File: src/text.transform.js [hunk 1]
```diff
@@ -1,8 +1,11 @@
...
+
+export function TextTransform(mimetype, value, document) {
   var el = document.createElement('pre')
-  el.textContent = data
+  var esc = ansi_up.escape_for_html(value)
+  el.innerHTML = ansi_up.ansi_to_html(esc)
   return el
 }
-TextTransform.mimetype = 'text/plain'
+TextTransform.mimetype = ['text/plain', 'jupyter/console-text']
```
Comment (rgbkrk) [inline src/text.transform.js]: :+1:

**Comment by rgbkrk** [comment]
To keep this backwards compatible, I'd make the old exports on the transforms you deleted point to the new unified one.

**Comment by lgeiger** [comment]
The `ConsoleText` transform was never included in transformime. You can still import it from `transformime-jupyter-transformers` as before. This shouldn't break backwards compatible. Or am I missing something?

**Comment by rgbkrk** [comment]
If you've got it covered, by all means I trust in your judgement here.

**Pull request merged**